frontend/src/components/Rounting.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import '../App.global.scss';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import SignIn from '../screens/SignIn';
+import SignUp from '../screens/SignUp';
+import Home from '../screens/Home';
+import Topics from '../screens/Topics';
+import MyTeam from '../screens/MyTeam';
+import Profile from '../screens/Profile';
+import Nav from './Nav';
+import { useSelector, useDispatch } from 'react-redux';
+import { setLoginStatus } from '../state_manager/actions';
+
+function Rounting() {
+  const [loggedIn, setLoggedIn] = useState(false);
+
+  const dispatch = useDispatch();
+  // const loginStatus = useSelector(state => state.loginStatus);
```

Comment:
Why is this comment here? If you absolutely have to comment code, then I'd say have it as a function somewhere else (outside the normal flow of the program) and specify in a comment that it's not usable for now, or put another comment explaining why it's commented, perhaps a TODO statement?